### PR TITLE
Reduced Default Graphics Settings

### PR DESCRIPTION
**Pull request by himham-jak**
Per https://discord.com/channels/756287461377703987/756287461830688810/1018649195927838830
I just turned on particle culling by default, set msaa to 1 and moved some values to constants up top for easy editing. Let me know if I made any mistakes.

### CODE REVIEW

**Review by xTVaser — CHANGES_REQUESTED** [review]
Seems like you've intermixed swedish subtitle changes in with this PR -- things will fail because the files it references doesn't exist, you should pull those changes out into a separate PR.

File: goal_src/jak1/pc/pckernel-h.gc [hunk 1]
```diff
@@ -477,7 +478,7 @@
   "Set the default ps2 settings"
 
   (set! (-> obj ps2-read-speed?) #f)
-  (set! (-> obj ps2-parts?) #f)
+  (set! (-> obj ps2-parts?) #t)
```
Comment (xTVaser) [inline goal_src/jak1/pc/pckernel-h.gc:481]: what does `ps2-parts?` do?

Comment (ManDude) [reply goal_src/jak1/pc/pckernel-h.gc:481]: yes

**Comment by xTVaser** [comment]
Your experience for using git/github would be a lot better if you didn't do everything from the browser
- https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop

**Comment by himham-jak** [comment]
Ugh, github irks me. I'll comment out the swedish stuff

**Comment by himham-jak** [comment]
I do have the desktop client. I need to use it more.

**Comment by xTVaser** [comment]
Don't just comment it out, remove it entirely from the changeset.  The reason this is happening is because you aren't using a separate branch -- you're adding things to `master` (which is being used for this PR)

**Comment by himham-jak** [comment]
> Don't just comment it out, remove it entirely from the changeset. The reason this is happening is because you aren't using a separate branch -- you're adding more things to `master` (which is being used for this PR)

should be fixed. my bad

**Comment by xTVaser** [comment]
ok to test

**Review by xTVaser — APPROVED** [review]
Tested and seemed to work as expected -- I think it's better for the defaults to be conservative, though it's a little weird with the particle culling as everyone seems confused why it helps performance at all.

The new `defconstants` could probably be named a little better `DEFAULT_FRAME_RATE` instead of just `FRAME_RATE` for example but im not strongly opinionated on this.

**Pull request closed without merging**